### PR TITLE
fix: Handle input amounts based on the selected currency format

### PR DESCRIPTION
**Pull request by staylorwr**

### CODE REVIEW

File: src/validations.ts [hunk 1]
```diff
@@ -47,12 +54,14 @@ defineRule('validValidator', (validatorString: string) => {
   return !!safeAddress
 })
 
-defineRule('validAmount', (amountString: string) => {
```
Comment (caleboller) [inline src/validations.ts:50]: @staylorwr Comments to clean up?

File: src/views/Wallet/WalletStaking.vue [hunk 2]
```diff
@@ -215,6 +219,8 @@ const WalletStaking = defineComponent({
       unstakeTokens
     } = useWallet(router)
 
+    console.log(decimalType)
```
Comment (caleboller) [inline src/views/Wallet/WalletStaking.vue:222]: Console.log to remove.

**Review by caleboller — CHANGES_REQUESTED** [review]

**Review by artsycoder533 — APPROVED** [review]
Looks good to me!

File: src/views/Wallet/WalletTransaction.vue [hunk 3]
```diff
@@ -330,10 +327,18 @@ const WalletTransaction = defineComponent({
       }
     }
 
+    const blah = (evt: Event) => {
+      const value = (evt.target as HTMLInputElement).value
+      console.log(value)
+      values.amount = value.replace(/\d.,]/g, '')
+      console.log(values.amount)
+    }
+
```
Comment (artsycoder533) [inline src/views/Wallet/WalletTransaction.vue:336]: Did you mean to keep the blah function? lines 330-335

Comment (staylorwr) [reply src/views/Wallet/WalletTransaction.vue]: fixed!

**Review by caleboller — APPROVED** [review]

**Pull request merged**